apps/allocations/app/components/Shared/IdentityManager.js

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
```

Comment:
If the code for this file is the same that the one used on Projects, I think it should be better to put it on shared/ui folder and import from there at both app, since we are going to put extra effort to de-duplicate these kind of files

Reply:
There are two files, so it made sense to move them into separate folder:

shared/ui/components/identity/